src/features/dashboard/components/Dialogs/TokenCreationDialogSuccess/index.tsx

```diff
@@ -38,33 +36,40 @@ export const TokenCreationDialogSuccess = ({
   }, [tokens, lastTokenDisplayName]);
 
   return (
-    <Modal defaultOpen onOpenChange={onOpenChange}>
-      <Modal.Portal>
-        <div className='modal-overlay'>
-          <Modal.Overlay />
-          <Modal.PageContent className={styles.wrapper}>
-            <div className={styles.title}>Token created successfully!</div>
-            <div className={styles.modal}>
-              <p>
-                Please save this token key. For security reasons, it can&apos;t be viewed or copied
-                again. If you lose this key, you&apos;ll need to generate a new token.
-              </p>
-            </div>
-            <div className={styles.textField}>
-              <div>
-                <div className={styles.key}>Key</div>
-                {latestToken}
-              </div>
-              <CopyButton value={latestToken} has_admin={false} />
-            </div>
-            <div className={styles.button_wrapper}>
-              <Button color='primary' onClick={handleToggle} className={styles.btn}>
-                OK
-              </Button>
-            </div>
-          </Modal.PageContent>
+    <Modal
+      isOpened={true}
+      showHandleBar
+      disableCloseOnOverlay
+      isMobile={deviceType !== 'desktop'}
+      showPrimaryButton={true}
+      showSecondaryButton={false}
```

Comment:
do we really need this property ? can't we just skip `showSecondaryButton` property to hide button and if need primary button just add `showPrimaryButton` without `={true}` ?